src/components/look-controls.js

```diff
@@ -192,12 +207,18 @@ module.exports.Component = registerComponent('look-controls', {
     var yawObject = this.yawObject;
     var previousMouseEvent = this.previousMouseEvent;
 
-    if (!this.mouseDown || !this.data.enabled) { return; }
+    if (!this.data.enabled) {
```

Comment:
we need to add `if (!this.data.enabled) { return; }` checks to

* `onMouseDown`
* `onMouseMove`
* `releaseMouse`
* `releaseMouse`
* `onTouchStart`
* `onTouchMove`
* `onTouchEnd`
* `onPointerLockChange`
* `onPointerLockError`
* `onPointerLockChange`
* `onPointerLockError`